hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/net/NetworkTopology.java

```diff
@@ -176,7 +176,7 @@ Node chooseRandom(String scope, Collection<Node> excludedNodes,
    *
    * @return the chosen node
    */
-  Node chooseRandom(String scope, String excludedScope,
+  Node chooseRandom(String scope, List<String>  excludedScopes,
```

Comment:
I don't see this being used anywhere, should we remove it?

Reply:
OK